js/about/passwords.js

```diff
@@ -63,7 +63,7 @@ class SiteItem extends React.Component {
   }
 
   onDelete () {
-    aboutActions.deletePasswordSite(this.props.site)
+    aboutActions.deletePassword(this.props.site.toJS())
```

Comment:
sorry for the confusion, new `never save password` list will migrate and save in chromium as the password entry with `blacklisted_by_user = true` attribute.